components/studio-platform/plugins/org.wso2.integrationstudio.docker.distribution/src/org/wso2/integrationstudio/docker/distribution/action/MavenJobThread.java

```diff
@@ -103,7 +103,7 @@ private void executeMavenBuildJob() {
 					revertDockerConfigurationFile();
 					if (isBuildSuccess == 0 && authConfiguration != null && authConfiguration.isKubernetesProject()) {
 						executeMavenPushJob();
-					} else if (isBuildSuccess == 0 && authConfiguration == null) {
+					} else if (isBuildSuccess == 0 && !authConfiguration.isKubernetesProject()) {
```

Comment:
will this cause null pointer exception?

Reply:
No. Earlier there were no auth configuration used in the build operation. So authconfiguration==null is used to check whether it is a build process or build&push process. But now we are using authconfiguration for both operations. So, authconfiguration will not be null.